app/controllers/concerns/validate_move.rb

```diff
@@ -0,0 +1,15 @@
+module ValidateMove
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :validate_previous_id, only: :move
+  end
+
+  protected
+
+  def validate_previous_id
+    return true if params[:next_id].present? && params[:prev_id].blank?
```

Comment:
This doesn't work when moving a card to a list with no other cards since both `next_id` and `prev_id` is nil.